pkg/rhos/ocpgwdeployer.go

```diff
@@ -209,12 +220,14 @@ func (d *ocpGatewayDeployer) deployGWNode(gatewayCount int, groupName string,
 	return err
 }
 
-func (d *ocpGatewayDeployer) deployDedicatedGWNode(gatewayNodesToDeploy int, status reporter.Interface) error {
+func (d *ocpGatewayDeployer) deployDedicatedGWNode(gatewayNodesToDeploy int, addInternalSG bool,
+	status reporter.Interface,
+) error {
 	for i := 0; i < gatewayNodesToDeploy; i++ {
 		gwNodeName := d.InfraID + "-submariner-gw" + strconv.Itoa(i)
 		status.Start("Deploying dedicated Submariner gateway node %s", gwNodeName)
 
-		err := d.deployGateway(strconv.Itoa(i))
```

Comment:
Using "strconv.Itoa(i)" was probably masking the issue that I pointed to above. But once we change it to use UUID in the Yaml, I feel we will run into the problem more often.